packages/devextreme/js/ui/chat.d.ts

```diff
@@ -0,0 +1,182 @@
+import Widget, { WidgetOptions } from './widget/ui.widget';
+import {
+    Cancelable,
+    EventInfo,
+    NativeEventInfo,
+    InitializedEventInfo,
+    ChangedOptionInfo,
+} from '../events/index';
+
+/**
+ * @docid _ui_chat_ContentReadyEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type ContentReadyEvent = EventInfo<dxChat>;
```

Comment:
I think we don't need contentReady in Chat

Reply:
where vanilla JS users will apply options at runtime after a component is rendered\refreshed in this component?

Reply:
At this time, we have decided not to publish this in the documentation